### PR TITLE
Add docs for installation of NF edge releases

### PR DESCRIPTION
**Pull request by drpatelh**
Signed-off-by: Harshil Patel <drpatelhh@gmail.com>

### CODE REVIEW

File: docs/getstarted.rst [hunk 1]
```diff
@@ -41,10 +41,31 @@ It only needs two easy steps:
...
+============
+
+Stable releases will be becoming more infrequent as `Nextflow` shifts its development model to becoming more dynamic via the usage of plugins. 
+This will allow functionality to be added as an extension to the core codebase with a release cycle that could potentially be independent to that of `Nextflow` itself. 
+As a result of the reduction in stable releases, you may be required to use `Nextflow` ``edge`` releases in order to be able to exploit cutting "edge" features.
+
+There are a number of ways you can install `Nextflow` ``edge`` releases, the main difference with stable releases being that you have to ``export`` the version you would like to install before issuing the appropriate installation/execution commands as highlighted below:
+
+#.  If you have `Nextflow` installed already, you can issue the version you would like to use on the same line as the pipeline command and it will be fetched if required before the pipeline execution:
+    ``NXF_VER="20.11.0-edge" nextflow run tutorial.nf``.
```
Comment (abhi18av) [inline docs/getstarted.rst:60]: Perhaps we should use the latest `edge` release as an example?

Comment (drpatelh) [reply docs/getstarted.rst]: Absolutely! Good idea :) Fixed https://github.com/nextflow-io/nextflow/pull/2077/commits/e5d762aef5e02618c51a663b568c426689fd4ab4

**Comment by drpatelh** [comment]
I hope the file renders properly in the main NF docs 😅 I tried looking at it with various previewers and they all looked different.

**Comment by pditommaso** [comment]
Ok, merged manually with a radical review .. 21eba5d1. Hope you won't mind 😅 . thanks!

**Comment by drpatelh** [comment]
Alt text

**Pull request closed without merging**